adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -15,9 +15,38 @@
  * limitations under the License.
  */
 
-(function( window, undefined ) {
-
-    var Keycloak = function (config) {
+(function(root, factory) {
+    if (typeof exports === 'object' && typeof module === 'object')
```

Comment:
> This means that even if AMD is present the Keycloak global is still exposed

Good point! Will update it to be part of window as well.

Mhh yeah, you are correct. I often forget that you have to actually create these files with AMD. I'll update the code so it will pass these dependencies from widow.